src/main/java/org/rust/lang/core/psi/RustTokenElementTypes.java

```diff
@@ -95,61 +96,60 @@
-    RustTokenType MUL = new RustTokenType("*");
-    RustTokenType MULEQ = new RustTokenType("*=");
-    RustTokenType OR = new RustTokenType("|");
-    RustTokenType OREQ = new RustTokenType("|=");
-    RustTokenType PLUS = new RustTokenType("+");
-    RustTokenType PLUSEQ = new RustTokenType("+=");
-    RustTokenType RBRACE = new RustTokenType("}");
-    RustTokenType RBRACK = new RustTokenType("]");
-    RustTokenType REM = new RustTokenType("%");
-    RustTokenType REMEQ = new RustTokenType("%=");
-    RustTokenType RPAREN = new RustTokenType(")");
-    RustTokenType SEMICOLON = new RustTokenType(";");
-    RustTokenType XOR = new RustTokenType("^");
-    RustTokenType XOREQ = new RustTokenType("^=");
-    RustTokenType Q = new RustTokenType("?");
-    RustTokenType AT = new RustTokenType("@");
-    RustTokenType UNDERSCORE = new RustTokenType("_");
-    RustTokenType DOLLAR = new RustTokenType("$");
+    RustTokenType AND = new RustOperatorTokenType("&");
+    RustTokenType ANDEQ = new RustOperatorTokenType("&=");
+    RustTokenType ARROW = new RustOperatorTokenType("->");
+    RustTokenType FAT_ARROW = new RustOperatorTokenType("=>");
+    RustTokenType SHA = new RustOperatorTokenType("#");
+    RustTokenType COLON = new RustOperatorTokenType(":");
+    RustTokenType COLONCOLON = new RustOperatorTokenType("::");
+    RustTokenType COMMA = new RustOperatorTokenType(",");
+    RustTokenType DIV = new RustOperatorTokenType("/");
+    RustTokenType DIVEQ = new RustOperatorTokenType("/=");
+    RustTokenType DOT = new RustOperatorTokenType(".");
+    RustTokenType DOTDOT = new RustOperatorTokenType("..");
+    RustTokenType DOTDOTDOT = new RustOperatorTokenType("...");
+    RustTokenType EQ = new RustOperatorTokenType("=");
+    RustTokenType EQEQ = new RustOperatorTokenType("==");
+    RustTokenType EXCL = new RustOperatorTokenType("!");
+    RustTokenType EXCLEQ = new RustOperatorTokenType("!=");
+    RustTokenType GT = new RustOperatorTokenType(">");
+    RustTokenType LBRACE = new RustOperatorTokenType("{");
+    RustTokenType LBRACK = new RustOperatorTokenType("[");
+    RustTokenType LPAREN = new RustOperatorTokenType("(");
+    RustTokenType LT = new RustOperatorTokenType("<");
+    RustTokenType MINUS = new RustOperatorTokenType("-");
+    RustTokenType MINUSEQ = new RustOperatorTokenType("-=");
+    RustTokenType MUL = new RustOperatorTokenType("*");
+    RustTokenType MULEQ = new RustOperatorTokenType("*=");
+    RustTokenType OR = new RustOperatorTokenType("|");
+    RustTokenType OREQ = new RustOperatorTokenType("|=");
+    RustTokenType PLUS = new RustOperatorTokenType("+");
+    RustTokenType PLUSEQ = new RustOperatorTokenType("+=");
+    RustTokenType RBRACE = new RustOperatorTokenType("}");
+    RustTokenType RBRACK = new RustOperatorTokenType("]");
+    RustTokenType REM = new RustOperatorTokenType("%");
+    RustTokenType REMEQ = new RustOperatorTokenType("%=");
+    RustTokenType RPAREN = new RustOperatorTokenType(")");
```

Comment:
I think that `(`, `{`, `[` braces should not be colored as operators (they are not in Kotlin).